src/HomePage/StyledHomepage.ts

```diff
@@ -0,0 +1,11 @@
+import styled from "styled-components";
```

Comment:
change request: change name StyledHomepage=>StyledDivWrapper. Change and rename HomePage folder to lowercase (home) and move this folder to src/pages folder.